content/en/docs/policy-reports/openreports.md

```diff
@@ -0,0 +1,92 @@
+---
+title: OpenReports Integration 
+description: Use openreports.io/v1alpha1 for policy reporting in Kyverno.
+weight: 15
+---
+
+> **Note:** OpenReports integration is available as of Kyverno 1.15.
+
+Kyverno supports reporting policy results using the openreports.io/v1alpha1 API as an alternative to the default wgpolicyk8s reporting. This can be enabled using the `--openreportsEnabled` flag in the Kyverno controller.
```

Comment:
```suggestion
Kyverno supports reporting policy results using the `openreports.io/v1alpha1` API as an alternative to the default wgpolicyk8s reporting. This can be enabled using the `--openreportsEnabled` flag in the Kyverno controller.
```